packages/base-flambda/base-flambda.base/opam

```diff
@@ -0,0 +1,5 @@
+opam-version: "1.2"
+maintainer: "stephen.dolan@cl.cam.ac.uk"
+build: [["ocaml" "check_flambda.ml"]]
+available: [ ocaml-version >= "4.03" ]
```

Comment:
I think it'd be good to make the flambda dependency explicit in the constraints by enumerating the flambda compilers here, even though there are a few of them.  I think the important ones at the moment are

* 4.06.0+trunk+flambda
* 4.06.0+trunk+fp+flambda
* 4.05.0+flambda (not yet available)
* 4.05.0+fp+flambda (not yet available)
* 4.04.0+flambda
* 4.04.0+fp+flambda
* 4.03.0+flambda
* 4.03.0+fp+flambda

The others are mostly obsolete `beta` and `trunk` versions.